src/apprt/gtk/Window.zig

```diff
@@ -328,53 +355,89 @@ pub fn updateConfig(
     self: *Window,
     config: *const configpkg.Config,
 ) !void {
-    self.winproto.updateConfigEvent(config) catch |err| {
-        // We want to continue attempting to make the other config
-        // changes necessary so we just log the error and continue.
-        log.warn("failed to update window protocol config error={}", .{err});
-    };
+    self.config = DerivedConfig.init(config);
```

Comment:
Copying over is fine for now, but we may need to make this a little more sophisticated if we ever need to alloc parts of the derived config.